diff --git a/adev/src/app/core/services/header.service.ts b/adev/src/app/core/services/header.service.ts
index a1557c857f..0f7ce89938 100644
--- a/adev/src/app/core/services/header.service.ts
+++ b/adev/src/app/core/services/header.service.ts
@@ -3,13 +3,13 @@
  * Copyright Google LLC All Rights Reserved.
  *
  * Use of this source code is governed by an MIT-style license that can be
- * found in the LICENSE file at https://angular.dev/license
+ * found in the LICENSE file at https://angular.jp/license
  */
 
 import {DOCUMENT} from '@angular/common';
 import {inject, Service} from '@angular/core';
 
-const ANGULAR_DEV = 'https://angular.dev';
+const ANGULAR_DEV = 'https://angular.jp';
 
 /**
  * Information about the deployment of this application.
