diff --git a/adev/src/app/core/constants/versions.ts b/adev/src/app/core/constants/versions.ts
index fcb60cbd77..f59ab93291 100644
--- a/adev/src/app/core/constants/versions.ts
+++ b/adev/src/app/core/constants/versions.ts
@@ -7,11 +7,11 @@
  */
 
 export const VERSIONS_CONFIG = {
-  aDevVersionsLinkPattern: 'https://{{prefix}}{{version}}angular.dev',
+  aDevVersionsLinkPattern: 'https://{{prefix}}{{version}}angular.jp',
   aioVersions: [
     {
       version: 'v17',
-      url: 'https://v17.angular.io/docs',
+      url: 'https://v17.angular.jp/docs',
     },
     {
       version: 'v16',
diff --git a/adev/src/app/core/services/version-manager.service.ts b/adev/src/app/core/services/version-manager.service.ts
index 4b0fcce335..eb93692be2 100644
--- a/adev/src/app/core/services/version-manager.service.ts
+++ b/adev/src/app/core/services/version-manager.service.ts
@@ -56,11 +56,11 @@ export class VersionManager {
   // List of Angular Docs versions which includes current version, next and rc.
   private getRecentVersions(): Version[] {
     return [
-      {
-        url: this.getAdevDocsUrl('next'),
-        displayName: `next`,
-        version: 'next',
-      },
+      // {
+      //   url: this.getAdevDocsUrl('next'),
+      //   displayName: `next`,
+      //   version: 'next',
+      // },
       // Note: 'rc' should not be visible for now
       // {
       //   url: this.getAdevDocsUrl('rc'),
