diff --git a/adev/src/app/app.config.ts b/adev/src/app/app.config.ts
index 1bc572a030..40345f5292 100644
--- a/adev/src/app/app.config.ts
+++ b/adev/src/app/app.config.ts
@@ -77,7 +77,7 @@ export const appConfig: ApplicationConfig = {
     provideAnimationsAsync(),
     {
       provide: CURRENT_MAJOR_VERSION,
-      useValue: Number(VERSION.major),
+      useValue: 19,
     },
     {provide: ENVIRONMENT, useValue: environment},
     {
