diff --git a/aio/src/app/search/search.worker.ts b/aio/src/app/search/search.worker.ts
index f1c44bc30b..131f976a08 100644
--- a/aio/src/app/search/search.worker.ts
+++ b/aio/src/app/search/search.worker.ts
@@ -1,6 +1,11 @@
 /// <reference lib="webworker" />
 import * as lunr from 'lunr';
 
+declare const require: any;
+require('lunr-languages/lunr.stemmer.support.js')(lunr);
+require('lunr-languages/tinyseg.js')(lunr);
+require('lunr-languages/lunr.ja.js')(lunr);
+require('lunr-languages/lunr.multi.js')(lunr);
 import {WebWorkerMessage} from '../shared/web-worker-message';
 
 const SEARCH_TERMS_URL = '/generated/docs/app/search-data.json';
@@ -45,6 +50,7 @@ function createIndex(loadIndexFn: IndexLoader): lunr.Index {
   const queryLexer = (lunr as any as {QueryLexer: {termSeparator: RegExp}}).QueryLexer;
   queryLexer.termSeparator = lunr.tokenizer.separator = /\s+/;
   return lunr(function() {
+    this.use((lunr as any).multiLanguage('en', 'ja'));
     this.pipeline.remove(lunr.stemmer);
     this.ref('path');
     this.field('topics', {boost: 15});
diff --git a/aio/tools/transforms/angular-base-package/processors/generateKeywords.js b/aio/tools/transforms/angular-base-package/processors/generateKeywords.js
index dfe430c705..888c07a3f2 100644
--- a/aio/tools/transforms/angular-base-package/processors/generateKeywords.js
+++ b/aio/tools/transforms/angular-base-package/processors/generateKeywords.js
@@ -126,11 +126,11 @@ module.exports = function generateKeywordsProcessor(log) {
           token = token.trim();
 
           // Trim unwanted trivia characters from the start and end of the token
-          const TRIVIA_CHARS = '[\\s_"\'`({[<$*)}\\]>.,-]';
+          const TRIVIA_CHARS = '[\\s_"\'`({[<$*)}\\]>.,、。-]';
           // Tokens can contain letters, numbers, underscore, dot or hyphen but not at the start or end.
           // The leading TRIVIA_CHARS will capture any leading `.`, '-`' or `_` so we don't have to avoid them in this regular expression.
           // But we do need to ensure we don't capture the at the end of the token.
-          const POSSIBLE_TOKEN = '[a-z0-9_.-]*[a-z0-9]';
+          const POSSIBLE_TOKEN = '([a-z0-9_.-]*[a-z0-9]|[^\x00-\x7F]+)';
           token = token.replace(new RegExp(`^${TRIVIA_CHARS}*(${POSSIBLE_TOKEN})${TRIVIA_CHARS}*$`, 'i'), '$1');
 
           // Skip if blank or in the ignored words list
diff --git a/aio/package.json b/aio/package.json
index a3b0fedf9a..73a3209536 100644
--- a/aio/package.json
+++ b/aio/package.json
@@ -144,6 +144,7 @@
     "lighthouse-logger": "^1.3.0",
     "lodash": "^4.17.21",
     "lunr": "^2.3.9",
+    "lunr-languages": "1.10.0",
     "npm-run-all": "^4.1.5",
     "protractor": "~7.0.0",
     "puppeteer-core": "19.5.2",
diff --git a/aio/yarn.lock b/aio/yarn.lock
index 1c43d5b323..68e7f301ea 100644
--- a/aio/yarn.lock
+++ b/aio/yarn.lock
@@ -9913,6 +9913,11 @@ lru_map@^0.3.3:
   resolved "https://registry.yarnpkg.com/lru_map/-/lru_map-0.3.3.tgz#b5c8351b9464cbd750335a79650a0ec0e56118dd"
   integrity sha512-Pn9cox5CsMYngeDbmChANltQl+5pi6XmTrraMSzhPmMBbmgcxmqWry0U3PGapCU1yB4/LqCcom7qhHZiF/jGfQ==
 
+lunr-languages@1.10.0:
+  version "1.10.0"
+  resolved "https://registry.yarnpkg.com/lunr-languages/-/lunr-languages-1.10.0.tgz#2afe9fff47b435d9bc74bd372fb923dbf8ee1990"
+  integrity sha512-BBjKKcwrieJlzwwc9M5H/MRXGJ2qyOSDx/NXYiwkuKjiLOOoouh0WsDzeqcLoUWcX31y7i8sb8IgsZKObdUCkw==
+
 lunr@^2.3.9:
   version "2.3.9"
   resolved "https://registry.yarnpkg.com/lunr/-/lunr-2.3.9.tgz#18b123142832337dd6e964df1a5a7707b25d35e1"
diff --git a/aio/BUILD.bazel b/aio/BUILD.bazel
index d67502afae..32972bf625 100644
--- a/aio/BUILD.bazel
+++ b/aio/BUILD.bazel
@@ -168,6 +168,7 @@ APPLICATION_DEPS = [
     "@aio_npm//@types/lunr",
     "@aio_npm//@types/trusted-types",
     "@aio_npm//lunr",
+    "@aio_npm//lunr-languages",
     "@aio_npm//rxjs",
     "@aio_npm//safevalues",
     "@aio_npm//tslib",
